### PR TITLE
chore: add the ability to configure timezone

### PR DESCRIPTION
**Pull request by ono-max**
Close https://github.com/bucketeer-io/bucketeer/issues/323

### CODE REVIEW

File: pkg/eventpersister/persister/persister.go [hunk 1]
```diff
@@ -113,11 +113,13 @@ type Persister struct {
 	cancel                func()
 	doneCh                chan struct{}
 	evaluationCountCacher cache.MultiGetDeleteCountCache
+	location              *time.Location
```
[the remark this answers is not in the document]
Comment (ono-max) [reply pkg/eventpersister/persister/persister.go:116]: We'll remove `newEvaluationCountkey` method after https://github.com/bucketeer-io/bucketeer/pull/332 is released.
Thus, we don't have to care about it.

[the remark this answers is not in the document]
Comment (ono-max) [reply pkg/eventpersister/persister/persister.go]: Thanks, I noticed that I mistakenly added `location`. I removed it.

**Review by masaaania — APPROVED** [review]
LGTM!

**Review by cre8ivejp — APPROVED** [review]
👍

**Pull request merged**